tests/integration/test_otlp_logs.py

```diff
@@ -168,6 +168,7 @@ def test_otlp_logs_conversion(
             "projectId": "42",
             "received": time_within_delta(),
             "retentionDays": 30,
+            "downsampledRetentionDays": 390,
```

Comment:
I assume it's not a problem for Logs if Relay writes this field now?

Reply:
I think that's because of the upgrade of `sentry-protos`. Kevan mentioned this here: https://github.com/getsentry/relay/pull/5207#discussion_r2404628437